docs/diagrams/ListReviewActivityDiagram.puml

```diff
@@ -5,7 +5,7 @@ start
 'Since the beta syntax does not support placing the condition outside the
 'diamond we place it as the true branch instead.
 if () then ([Command is Valid])
-    :FoodWhere lists all [Review]s;
+    :FoodWhere lists all [Reviews];
```

Comment:
Maybe we should discuss what format to use for this, but initially I would have thought [Review] is the name of the class.

Reply:
Hmm.. in this case I will just revert to the original formatting then.